[Doc Modernization] Redirect links updates

v-kaviyas opened the pull request:
Updated Partner center link for the URL - Teams app usage report.

acrolinxatmsft1 commented:
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/design/overview-analytics.md](https://github.com/MicrosoftDocs/msteams-docs/blob/3ced5a49993a4d9b3db002900658212e23ca5849/msteams-platform%2Fconcepts%2Fdesign%2Foverview-analytics.md) | 82 | 84 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/46323ff2-08d0-49df-b79e-9058ad0dbc26) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

acrolinxatmsft1 commented:
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/design/overview-analytics.md](https://github.com/MicrosoftDocs/msteams-docs/blob/5ba9b2c1b49e1065c1456b2bfd6b7b80b9e61ba0/msteams-platform%2Fconcepts%2Fdesign%2Foverview-analytics.md) | 82 | 84 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/58bc5050-8693-48a9-bc6b-395f25d90489) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

acrolinxatmsft1 commented:
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/design/designing-apps-in-meetings.md](https://github.com/MicrosoftDocs/msteams-docs/blob/be7926b684a99097ce0128afaf15a404600affa2/msteams-platform%2Fapps-in-teams-meetings%2Fdesign%2Fdesigning-apps-in-meetings.md) | 93 | 21 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/f291730f-7b04-4ad8-b19d-5c02de3a9f7b) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/concepts/design/overview-analytics.md](https://github.com/MicrosoftDocs/msteams-docs/blob/be7926b684a99097ce0128afaf15a404600affa2/msteams-platform%2Fconcepts%2Fdesign%2Foverview-analytics.md) | 82 | 84 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/e1a695cc-b690-468e-9600-e4b669b93861) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

acrolinxatmsft1 commented:
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/design/designing-apps-in-meetings.md](https://github.com/MicrosoftDocs/msteams-docs/blob/1f1634f59498b356b6ba1e26c608c38c7fe31a9a/msteams-platform%2Fapps-in-teams-meetings%2Fdesign%2Fdesigning-apps-in-meetings.md) | 93 | 21 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/92015b0e-cc6a-4809-b965-1f6cef34d8ab) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/concepts/design/overview-analytics.md](https://github.com/MicrosoftDocs/msteams-docs/blob/1f1634f59498b356b6ba1e26c608c38c7fe31a9a/msteams-platform%2Fconcepts%2Fdesign%2Foverview-analytics.md) | 82 | 84 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/f74f3d9a-db02-4a93-829f-ed9f0509a191) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

acrolinxatmsft1 commented:
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/design/designing-apps-in-meetings.md](https://github.com/MicrosoftDocs/msteams-docs/blob/b43134a5fb555394db356fb2318b9a1faef645cf/msteams-platform%2Fapps-in-teams-meetings%2Fdesign%2Fdesigning-apps-in-meetings.md) | 93 | 21 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/214bd32c-ab7f-44c7-b772-9bf5e1377446) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/concepts/design/overview-analytics.md](https://github.com/MicrosoftDocs/msteams-docs/blob/b43134a5fb555394db356fb2318b9a1faef645cf/msteams-platform%2Fconcepts%2Fdesign%2Foverview-analytics.md) | 82 | 84 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/d432f4dd-f2c6-42de-a44c-89c11c102cb2) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

acrolinxatmsft1 commented:
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/design/designing-apps-in-meetings.md](https://github.com/MicrosoftDocs/msteams-docs/blob/75dec36f94ea9c54e44a5c4ac31389588a54d23c/msteams-platform%2Fapps-in-teams-meetings%2Fdesign%2Fdesigning-apps-in-meetings.md) | 93 | 21 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/ac626c19-d4fa-4132-ad5a-bbac2af3fa56) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/apps-in-teams-meetings/teams-apps-in-meetings.md](https://github.com/MicrosoftDocs/msteams-docs/blob/75dec36f94ea9c54e44a5c4ac31389588a54d23c/msteams-platform%2Fapps-in-teams-meetings%2Fteams-apps-in-meetings.md) | 90 | 20 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/dbc0dee4-1d77-4440-929a-462488936113) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/concepts/design/overview-analytics.md](https://github.com/MicrosoftDocs/msteams-docs/blob/75dec36f94ea9c54e44a5c4ac31389588a54d23c/msteams-platform%2Fconcepts%2Fdesign%2Foverview-analytics.md) | 82 | 84 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/6c967e5a-9e37-4d52-a842-7c4339f04565) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

v-kaviyas commented:
Updated 2 links.

Updated Teams app usage report in [App Analytics - Overview - Teams | Microsoft Learn](https://review.learn.microsoft.com/en-us/microsoftteams/platform/concepts/design/overview-analytics?branch=pr-en-us-13060) 
(The previous link navigates to "Maintain your published Microsoft Teams app").
Updated Configure your app for meetings in [Designing your meeting extension - Teams | Microsoft Learn](https://learn.microsoft.com/en-us/microsoftteams/platform/apps-in-teams-meetings/design/designing-apps-in-meetings?view=msteams-client-js-latest)
(The previous link navigated to "Build tabs for meeting").

acrolinxatmsft1 commented:
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/design/designing-apps-in-meetings.md](https://github.com/MicrosoftDocs/msteams-docs/blob/1c746fea3dccaeeee424912b608235e5de7bc264/msteams-platform%2Fapps-in-teams-meetings%2Fdesign%2Fdesigning-apps-in-meetings.md) | 93 | 21 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/226ee34f-f653-4164-a875-4414f7e64386) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/apps-in-teams-meetings/teams-apps-in-meetings.md](https://github.com/MicrosoftDocs/msteams-docs/blob/1c746fea3dccaeeee424912b608235e5de7bc264/msteams-platform%2Fapps-in-teams-meetings%2Fteams-apps-in-meetings.md) | 90 | 20 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/6485f547-2efe-4511-9182-42db6c53fe2b) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/concepts/design/overview-analytics.md](https://github.com/MicrosoftDocs/msteams-docs/blob/1c746fea3dccaeeee424912b608235e5de7bc264/msteams-platform%2Fconcepts%2Fdesign%2Foverview-analytics.md) | 82 | 84 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/ed02458b-4d67-40cc-b7ac-82064ef10ed3) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).
